easybuild/easyconfigs/g/GULP/gulp-6.0-mkgulp.patch

```diff
@@ -0,0 +1,57 @@
+diff --git a/gulp-6.0.org/Src/mkgulp b/gulp-6.0/Src/mkgulp
+index 28afb80..84549b0 100755
+--- a/gulp-6.0.org/Src/mkgulp
++++ b/gulp-6.0/Src/mkgulp
+@@ -170,7 +170,7 @@ fi
+ #  Include file directories
+ #
+ #--USER--Start
+-echo 'INC=-I.. -I/usr/local/include/' >> makefile
++echo 'INC=-I.. -I${EBROOTFFTW}/include -I/usr/local/include/' >> makefile
+ #--USER--End
+ #  If using Macports then use the following instead
+ #--USER--Start
+@@ -202,19 +202,19 @@ case $compiler in
+                echo 'SLIBS= ' >> makefile
+ #  If there are optimised maths libraries available replace the routines below
+ #--USER--Start
+-               echo 'MLIBS=lapack.o blas.o' >> makefile
++               echo 'MLIBS=-L${LD_LIBRARY_APTH} -lscalapack -lopenblas ' >> makefile
```

Comment:
Typo "APTH", you can't use LD_LIBRARY_PATH for this since -L doesn't understand : lists, and it should be LIBRARY_PATH if anything but it would still not work. We have a better flag that doesn't hardcode "openblas".
`SCALAPACK_LIB_DIR`, `LIBSCALAPACK` , `LIBBLAS` should suffice here.